### ISSUE TITLE
Error with Cocoapods

### ISSUE THREAD

**Issue by RayanSaeed** [Yalantis/Koloda#55]
Hi, I am getting this error. This happened to me after updating to Xcode 7.1

Right now I have the latest of everything, Xcode, OS X El Capitan and Cocoapods.. 
All of them are latest. What is the reason behind this? and how to fix it?

![eba21578-62c2-11e5-8783-55e0793604cb](https://cloud.githubusercontent.com/assets/8598858/10696513/d30ebcee-79c1-11e5-9eb7-34fbeaa3f541.png)

**Comment by grimabe** [comment]
There is a CocoaPod issue for this but no solution for now : https://github.com/CocoaPods/CocoaPods/issues/4420

**Comment by danramteke** [comment]
Oh hey, just saw this after commenting on CocoaPods/CocoaPods#4420

Here's a workaround for now. I added this to my Podfile, and my project is good to go!
```
post_install do |installer|
    `find Pods -regex 'Pods/pop.*\\.h' -print0 | xargs -0 sed -i '' 's/\\(<\\)pop\\/\\(.*\\)\\(>\\)/\\"\\2\\"/'`
end
```

**Comment by grimabe** [comment]
+1 it is working

**Comment by skingtree** [comment]
@danramteke thx, it's working +1.

**Comment by AEugene** [comment]
@danramteke Thanks. Could you help to integrate this solution into podspec?

**Comment by danramteke** [comment]
@AEugene I believe that this is a temporary workaround. It looks like the Cocoapods team is aware of this issue affecting multiple libraries. For that reason, I recommend waiting for the Cocoapods team to release a fix.

**Comment by iliubave** [comment]
@danramteke Thanks a lot!

**Comment by lionello** [comment]
We should commit this workaround to ensure the example works out of the box. Don't let people waste time tracking down the solution to this.

**Comment by AEugene** [comment]
Issue will be fixed with CocoaPods/CocoaPods#4420
